Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -795,47 +774,22 @@ - (UIView *)loadViewForMediaAlbum
     self.unloadCellContentBlock = ^{
         [albumView unloadMedia];
     };
-    for (UIView *itemView in albumView.itemViews) {
-        OWSBubbleShapeView *strokeView = [[OWSBubbleShapeView alloc] initDraw];
-        strokeView.strokeColor = [UIColor colorWithWhite:0.5f alpha:0.4f];
-        strokeView.strokeThickness = 1.f;
-        [itemView addSubview:strokeView];
-        [self.bubbleView addPartnerView:strokeView];
-        [self.viewConstraints addObjectsFromArray:[strokeView ows_autoPinToSuperviewEdges]];
+
+    // Only apply "inner shadow" for single media, not albums.
```

Comment:
I thought the spec had shadows in albums too, was that an intentional design change?

Reply:
I couldn't find out a good way to do it, so Myles approved taking them out.